### ISSUE TITLE
Wiki section

### ISSUE THREAD

**Issue by Turbotailz** [Turbotailz/LuckPermsWeb#29]
Mirror of the github wiki, converting each page to a Vue component/route to allow for instant page loads and other customisations. 

Look into using webpack-contrib/raw-loader and markdown-it/markdown-it for .md loading.

**Comment by Andre601** [comment]
That would be neat...
Wouldn't VuePress be already enough tho?

**Comment by Turbotailz** [comment]
Yeah probably, but I am not sure how it will play with my current setup.
